aiida/tools/importexport/dbexport/__init__.py

```diff
@@ -420,13 +638,21 @@ def export_tree(
 
         progress_bar.update()
 
-    # We will iteratively explore the AiiDA graph to find further nodes that should also be exported.
-    # At the same time, we will create the links_uuid list of dicts to be exported
+    return entities_starting_set, given_node_entry_ids
+
+
+def _collect_export_nodes(given_node_entry_ids: Set[int], silent: bool,
```

Comment:
```suggestion
def _collect_node_ids(given_node_entry_ids: Set[int], silent: bool,
```

Reply:
fixed in b689f5e6986c15ab18ad225e5659f1ef3bf8f9c0